### PR TITLE
CCM-1620: Post test failures to teams

### PR DESCRIPTION
**Pull request by Kennedy1265**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Comment by Kennedy1265** [comment]
image

File: azure/nightly-int-qa-pipeline.yml [hunk 1]
```diff
@@ -26,9 +26,15 @@ variables:
...
+steps:
+  - template: ./templates/run-tests.yml
+    parameters:
+      nightly: true
+      environment: 'internal-qa'
+      test_command: 'make internal-qa-test'
+  - bash: |
+      curl -X POST $(ALERTS_PROD_API_WEBHOOK_URI) \
+      -H 'Content-Type: application/json' \
+      -d '{"@type":"MessageCard","@context":"http://schema.org/extensions","themeColor":"ff0000","title":"Nightly Internal QA Test Run Failed","text":"https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId='$(build.buildid)'&view=results"}'
```
Comment (PhilipSkinner) [inline azure/nightly-int-qa-pipeline.yml:38]: I wonder if you want to template this alerting so we don't replicate the action for sending the message?

**Review by PhilipSkinner — CHANGES_REQUESTED** [review]
Please see previous comments

**Comment by Kennedy1265** [comment]
closing due to further testing

**Pull request closed without merging**